Duplicati/Library/Main/Operation/Backup/BackendUploader.cs

```diff
@@ -369,28 +380,29 @@ private async Task DoPut(FileEntryItem item, IBackend backend, CancellationToken
             await m_database.CommitTransactionAsync("CommitAfterUpload");
         }
 
-        private void HandleProgress(ThrottledStream ts, long pg)
+        private void HandleProgress(ThrottledStream ts, long progress, string path)
         {
             if (!m_taskreader.TransferProgressAsync.WaitForTask().Result)
                 throw new OperationCanceledException();
 
-            // Update the throttle speeds if they have changed
-            string tmp;
-            m_options.RawOptions.TryGetValue("throttle-upload", out tmp);
+            var updateThrottleSpeeds = false;
+            m_options.RawOptions.TryGetValue("throttle-upload", out var tmp);

```

Comment:
Is it possible to use `m_options.MaxUploadPrSecond` here?  I would like to reduce the number of string references to options and use the named properties when feasible.

Reply:
That makes sense. I'll get that done in my next commit.